lib/shopify_api/resources/product.rb

```diff
@@ -16,6 +16,15 @@ def price_range
       end
     end
 
+    def serializable_hash(options = {})
+      super.except("total_inventory")
```

Comment:
I'm confused why this is needed? For versions *without* the field, it won't be here anyway. And for versions *with* it, this is a breaking change since it's done unconditionally.

If someone were still on 2019-07 for example, this would be a breaking change.

Same question applies for the variant change below.

Reply:
Right, I see what you mean. Honestly, I can't remember why we added that (it was before Christmas) but looking at it now and based on your arguments above I agree that we can remove it. 👍 

@eljojo Do you remember the logic behind the changes in `serializable_hash`?

Reply:
I think in these scenarios we can actually strip them out from `serializable_hash` in versions `2019-10` and later; if we already know we're going to fail those requests without clients making those changes, we should at least make it easier for them by omitting them in the scenario where they copy the response directly into the request.